tests/robottelo/test_func_locker.py

```diff
@@ -0,0 +1,321 @@
+# coding: utf-8
```

Comment:
I'm curious what's the difference between `# coding: utf-8` and `# -*- encoding: utf-8 -*-` if there's any. Most of our files use second format.